pkg/operation/botanist/component/kubeapiserverexposure/kube_apiserver_sni.go

```diff
@@ -94,10 +98,12 @@ type envoyFilterTemplateValues struct {
 
 func (s *sni) Deploy(ctx context.Context) error {
 	var (
-		destinationRule = s.emptyDestinationRule()
-		envoyFilter     = s.emptyEnvoyFilter()
-		gateway         = s.emptyGateway()
-		virtualService  = s.emptyVirtualService()
+		destinationRule        = s.emptyDestinationRule()
+		envoyFilter            = s.emptyEnvoyFilter()
+		gateway                = s.emptyGateway()
+		virtualService         = s.emptyVirtualService()
+		accessControlAPIServer = s.emptyAuthorizationPolicyApiServer()
```

Comment:
To keep the names more aligned:
```suggestion
		authorizationPolicyApiServer = s.emptyAuthorizationPolicyApiServer()
```